From 265a7ff10fb4380c64ed03a6ee08f828d62a16e1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mika=20Bostr=C3=B6m?= <bostik@odin.bostik.iki.fi>
Date: Wed, 19 Dec 2012 15:01:00 +0200
Subject: [PATCH 08/10] Drop large webkit components from build

Take out ANGLE, 3D, WebGL; none of those are required for now.
---
 Tools/qmake/mkspecs/features/features.prf |   10 ----------
 1 file changed, 10 deletions(-)

diff --git a/Tools/qmake/mkspecs/features/features.prf b/Tools/qmake/mkspecs/features/features.prf
index 2a0eb40..28db53d 100644
--- a/Tools/qmake/mkspecs/features/features.prf
+++ b/Tools/qmake/mkspecs/features/features.prf
@@ -77,21 +77,11 @@ defineTest(detectFeatures) {
         WEBKIT_CONFIG += plugin_architecture_unsupported
     }
 
-    # Enable the USE(3D_GRAPHICS) flag when QtOpenGL is enabled.
-    # Disable on Windows CE for now, as ANGLE won't compile.
-    !wince*:contains(QT_CONFIG, opengl) {
-        # On Windows we require GLES2/EGL through ANGLE.
-        !win32|contains(QT_CONFIG, opengles2): WEBKIT_CONFIG += use_3d_graphics
-    }
-
     # Temporarily disable FTPDIR on Windows CE (missing functions from time.h)
     wince* {
         WEBKIT_CONFIG -= ftpdir
     }
 
-    # WebGL support - only enabled if we can use 3D graphics
-    use?(3d_graphics): WEBKIT_CONFIG += webgl
-
     # Enable CSS shaders when CSS filters and WebGL are enabled
     enable?(css_filters):enable?(webgl): WEBKIT_CONFIG += css_shaders
 
-- 
1.7.10.4

